launcher/sdw_updater_gui/Updater.py

```diff
@@ -406,24 +406,33 @@ def shutdown_and_start_vms():
     in case they are being used by another non-workstation VM.
     """
 
+    sdw_vms_in_order = [
+        "sys-whonix",
+        "sd-proxy",
+        "sd-whonix",
+        "sd-app",
```

Comment:
`sd-app` is dependant on `sd-proxy`, but not set as vm, but using a qrexec service. Its relative positioning is identical to the previous version, and didn't seem to pose a problem then, so fine as-is

Reply:
Good point, and given @eloquence's results, definitely worth revisiting. Not going to append any further changes here, since we've tested extensively on this diff, but will highlight your suggestion in #498 for follow-up.